### DISCUSSION TITLE
[CH6] const vs Symbol

### DISCUSSION THREAD

**Discussion by innerstella** [innerstella/ES2024#3] [Q&A] [answered]
둘 다 고유한 값을 가지는데, const와 Symbol의 차이가 무엇인가요?

**Comment by rlghks1490** [comment, marked as answer]
단순하게 차이점을 보자면 `const`는 상수를 선언하기 위해 사용되는 **키워드**이고, `Symbol`은 하나의 **타입**이다.

`const`는 수학의 파이와 같이 변하지 않는, 일반적으로 **매직 넘버**라 불리는 값들을 저장하기 위해 사용되고,
`Symbol`은 중복되지 않는 유일무이한 값이라는 점을 이용해 충돌 위험이 없는 **유일한 프로퍼티 키**를 만들기 위해 사용된다.
